pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```
@@ -278,6 +282,16 @@ public PgConnection(HostSpec[] hostSpecs,
       bindStringAsVarchar = true;
     }
 
+    // clobAsText?
+    if (PGProperty.CLOB_AS_TEXT.getBoolean(info)) {
+      this.clobAsText = true;
+    }
+
+    // blobAsBytea?
+    if (PGProperty.BLOB_AS_BYTEA.getBoolean(info)) {
+      this.blobAsBytea = true;
+    }
```

what happens if both of these are true ? I expect we should throw an error